common/app/views/fragments/nav/subSectionNav.scala.html

```diff
@@ -3,18 +3,19 @@
 @import common.{NewNavigation, Edition, LinkTo}
 @import views.support.RenderClasses
 
-<div class="tertiary-navigation">
+<ul class="tertiary-navigation">
     @defining(NewNavigation.SubSectionLinks.getSectionOrTagId(page)) { id =>
 
         @NewNavigation.SubSectionLinks.getSubSectionNavLinks(id, Edition(request), page.metadata.isFront).map { link =>
-
-            <a class="@RenderClasses(Map(
-                    "tertiary-navigation__link" -> true,
-                    "tertiary-navigation__link--current-section" -> (id == link.uniqueSection)
-                ))"
-                href="@LinkTo(link.url)" data-link-name="nav2 : tertiary : @{ if(link.longTitle.isEmpty) link.title else link.longTitle}">
-                    @link.title
-            </a>
+            <li class="tertiary-navigation__link-item">
+                <a class="@RenderClasses(Map(
+                        "tertiary-navigation__link" -> true,
```

Comment:
separate to this PR maybe, but we really should change `tertiary` to `subnav` to keep names consistent. My fault!